spaceship/lib/spaceship.rb

```diff
@@ -1,3 +1,4 @@
+require 'fastlane_core'
```

Comment:
Also not entirely sure here about the dependencies, although it should be less of a problem /cc @KrauseFx